cmake/kokkos_test_cxx_std.cmake

```diff
@@ -42,13 +42,16 @@ FUNCTION(kokkos_set_cxx_standard_feature standard)
   ELSEIF((KOKKOS_CXX_COMPILER_ID STREQUAL "NVIDIA") AND WIN32)
     MESSAGE(STATUS "Using no flag for C++${standard} standard as feature")
     GLOBAL_SET(KOKKOS_CXX_STANDARD_FEATURE "")
+  ELSEIF((KOKKOS_CXX_COMPILER_ID STREQUAL "Fujitsu"))
+    MESSAGE(STATUS "Using no flag for C++${standard} standard as feature")
+    GLOBAL_SET(KOKKOS_CXX_STANDARD_FEATURE "")
```

Comment:
How is this going to work for C++17 and later? Why isn't there a flag for this?

Reply:
because cmake doesnt' know the fujitsu compiler and hence can't figure out its standard flags and feature set. So we are lying to it here and work around this whole thing. And yes a C++ flag will still be added to the CXX flags during the actual build.

Reply:
Also what do I know about how cmake works, this magic weirdness apparently sidesteps some internal cmake settings/tests/stuff which can't handle situations it doesn't know about like the fujitsu compiler. This makes it work ...